app/controllers/candidate_interface/continuous_applications/course_choices/concerns/duplicate_course_redirect.rb

```diff
@@ -0,0 +1,24 @@
+module CandidateInterface
```

Comment:
Thoughts on havig this in `app/controllers/concerns/`?

Reply:
I'm not pushed, I'll let you decide.